projects/plugins/crm/src/automation/commons/triggers/events/class-event-new.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Jetpack CRM Automation Event_New trigger.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+/**
+ * Adds the Event_New class.
+ */
+class Event_New extends Base_Trigger {
+
+	/**
+	 * Get the slug name of the trigger.
+	 * @return string
+	 */
+	public static function get_slug(): string {
+		return 'jpcrm/event_new';
+	}
+
+	/**
+	 * Get the title of the trigger.
+	 * @return string
+	 */
+	public static function get_title(): ?string {
+		return __( 'New Event', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the description of the trigger.
+	 * @return string
+	 */
+	public static function get_description(): ?string {
+		return __( 'Triggered when a new event status is added', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the category of the trigger.
+	 * @return string
+	 */
+	public static function get_category(): ?string {
+		return __( 'event', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Listen to this trigger's target event.
+	 */
+	protected function listen_to_event() {
```

Comment:
If we're declaring `:string` etc for the rest, then we could add `:void` here

```suggestion
	protected function listen_to_event(): void {
```